students/hyska_monika/lesson_07_manipulating_strings_datetime_formatting/TablePrinter.py

```diff
@@ -0,0 +1,21 @@
+# function print table with each column right-justified
+
+from functions_lesson07 import print_table, find_max_sting
```

Comment:
If you are using those functions only here, then it's kind of strange to have a separate module for them.